lib/rspec/rails/example/system_example_group.rb

```diff
@@ -37,7 +37,7 @@ def method_name
         @method_name ||= [
           self.class.name.underscore,
           RSpec.current_example.description.underscore
-        ].join("_").tr(CHARS_TO_TRANSLATE.join, "_")[0...251] + "_#{rand(1000)}"
+        ].join("_").tr(CHARS_TO_TRANSLATE.join, "_")[0...100] + "_#{rand(1000)}"
```

Comment:
Can we have 200? It seems 50 chars would be enough for more added words, but 100 is going to be close to a normal spec files docstring length.